.github/workflows/build.yml

```
@@ -24,7 +24,7 @@ jobs:
       - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
 
       - name: Install Conan
-        run: pip3 install conan==1.48.2
+        run: pip3 install conan==1.51.0 --force
```

```suggestion
        run: pip3 install conan==${{ env.CONAN_VER }} --force
```

Can we variable-ize the version here and define it at the top of the workflow file since it appears in several spots.

---

done